assets/component-slider.css

```diff
@@ -47,6 +47,12 @@ slider-component.slider-component-full-width {
     margin-bottom: 1rem;
   }
 
+  .slider--mobile:after {
+    content: "";
+    width: 0;
+    padding-left: 1.5rem;
```

Comment:
<img src="https://screenshot.click/24-39-r69eq-vwgxv.png" width="400" /> 
This doesn't entirely fix the single-visible variant case. It seems to be receiving the gap value (I imagine because the are technically more items in the grid) and the 1.5rem from the pseudo padding.

Reply:
ill double check on it. It's supposed to calculate the spacing based on gap value too 🤔